package.json

```diff
@@ -6,7 +6,7 @@
     "test": "jest",
     "test:coverage": "npm run test -- --coverage",
     "test:watch": "jest --watch",
-    "storybook": "APP_BUILD_VERSION=1 APP_BINARY_VERSION=dev START_STORYBOOK=true expo start",
+    "storybook": "APP_BUILD_VERSION=1 APP_BINARY_VERSION=dev START_STORYBOOK=true MOCK=true expo start",
```

Comment:
Why do we need the extra MOCK?

Reply:
so that the logout screen can load on the Storybook without having the logout handlers run in the background and fail

Reply:
How come we cannot use `APP_BINARY_VERSION`? How are the rest of the components running on storybook without this MOCK attribute?

Reply:
Because the rest of the screens don't have any calls to services. They go through the `AuthStoreContextProvider` and `ConfigStoreContextProvider` which are not used in the storybook (which uses custom `AuthStoreContext.Provider`s). There's only one other screen that calls services which is `CampaignInitialisationScreen` however that's not shown in storybook